### PR TITLE
Add a doc about using Avo alongside existing tracking

### PR DESCRIPTION
**Pull request by alex-tpom6oh**

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝 Changed routes:


| [**…-and-existing-tracking**](https://docs-51fu3qlln.avo-dev.app/implementation/avo-and-existing-tracking) |
|:-:|
|<a href="https://docs-51fu3qlln.avo-dev.app/implementation/avo-and-existing-tracking">Screenshot of /implementation/avo-and-existing-tracking</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/ZmNyXAqDPn2Zw_JKpfQnMIq5Fd6z41O3qine0pU9s61kuvinapfBrDdYYj_DjhAvqWEslD2ABzj8fGCof7tz8A.png">(view full size)</a> |







Commit 3ac4411694d72efbd6d01efea0b5b897488238e2 (https://docs-51fu3qlln.avo-dev.app).

File: pages/implementation/avo-and-existing-tracking.mdx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+layout: 'guide'
+title: 'Avo Functions alongside existing tracking'
+---
+
+import Link from '../../components/Link';
+
+You can safely keep raw tracking calls in your code when adopting Avo codegen. In this case we recommend to migrate gradually, following 2 steps below.
```
Comment (hugihlynsson) [inline pages/implementation/avo-and-existing-tracking.mdx:8]: Should we stick to the _Functions_ branding here? 
```suggestion
You can safely keep raw tracking calls in your code when adopting Avo Functions. In this case we recommend to migrate gradually, following 2 steps below.
```

File: pages/implementation/avo-and-existing-tracking.mdx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+### 1. Start implementing all new tracking with Avo Functions.
+
+To do this you'd mark the new events you want to implement with Avo as "Implement with Avo" in the Avo workspace and they will appear as functions in the generated code.
```
Comment (hugihlynsson) [inline pages/implementation/avo-and-existing-tracking.mdx:12]:
```suggestion
Mark the new events you want to implement with Avo as "Implement with Avo" in the Avo workspace and they will appear as functions in the generated code.
```

File: pages/implementation/avo-and-existing-tracking.mdx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+<center>
+  <img
+    src={require('../../images/best-practices/avo-functions-alongside-existing-tracking/implement-with-avo-example.png')}
+    width="75%"
+    alt="Implement with Avo checkbox example"
+  />
+</center>
+
+Then you'd init the Avo functions with a  <Link passHref href="/implementation/destinations#destination-interface-callback-methods"><a>destination interface</a></Link> as usual.
```
Comment (hugihlynsson) [inline pages/implementation/avo-and-existing-tracking.mdx:22]:
```suggestion
Initiate the Avo functions with a  <Link passHref href="/implementation/destinations#destination-interface-callback-methods"><a>destination interface</a></Link>.
```

File: pages/implementation/avo-and-existing-tracking.mdx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+In the track methods of the destination interface (logEvent, identify etc.) you'd use your existing tracking SDK instance to track events.
```
Comment (hugihlynsson) [inline pages/implementation/avo-and-existing-tracking.mdx:24]:
```suggestion
In the track methods of the destination interface (logEvent, identify etc.) you use your existing tracking SDK instance to track events:
```

File: pages/implementation/avo-and-existing-tracking.mdx [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+With this one time setup the events you track with Avo functions will be sent to the same analytics destinations as your existing events.
+
+### 2. Migrate you existing implementation to Avo
+
+Going forward you'll be able to gradually swap old events implemented with raw analytics SDK with Avo implementation at your own pace by
+
+- marking more events as "Implement with Avo"
+- pulling the Avo code 
+- replacing existing SDK call with a call to Avo function
```
Comment (hugihlynsson) [inline pages/implementation/avo-and-existing-tracking.mdx:41]:
```suggestion
Going forward you'll be able to gradually swap old events implemented with raw analytics SDK with Avo implementation at your own pace by:

- Marking more events as "Implement with Avo"
- Pulling the Avo code 
- Replacing existing SDK call with a call your generated Avo Functions
```

File: pages/implementation/avo-and-existing-tracking.mdx [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+In the track methods of the destination interface (logEvent, identify etc.) you'd use your existing tracking SDK instance to track events.
+
+```pseudocode
+logEvent(eventName, eventProperties) {
+    // Log event in your destination
+    analytics.track(eventName, eventProperties)
+}
+```
+
+With this one time setup the events you track with Avo functions will be sent to the same analytics destinations as your existing events.
```
Comment (hugihlynsson) [inline pages/implementation/avo-and-existing-tracking.mdx:33]:
```suggestion
With this one time setup the events you track with Avo Functions will be sent to the same analytics destinations as your existing events.
```

**Review by hugihlynsson — APPROVED** [review]
Few suggestions on general copy improvements but overall it looks great

File: pages/implementation/avo-and-existing-tracking.mdx [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
+With this one time setup the events you track with Avo Functions will be sent to the same analytics destinations as your existing events.
+
+### 2. Migrate you existing implementation to Avo
+
+Going forward you'll be able to gradually swap old events implemented with raw analytics SDK with Avo implementation at your own pace by:
+
+- Marking more events as "Implement with Avo"
+- Pulling the Avo code 
+- Replacing existing SDK call with a call your generated Avo Functions
```
Comment (alex-tpom6oh) [inline pages/implementation/avo-and-existing-tracking.mdx:41]:
```suggestion
- Replacing existing SDK calls with calls of the generated Avo Functions
```

**Pull request merged**